google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2171,6 +2219,62 @@ Blob create(
    */
   Blob compose(ComposeRequest composeRequest);
 
+  /**
+   * Watches the changes on objects in the bucket.
+   *
+   * <p>Example of watchAll.
+   *
+   * <pre>{@code
+   * String bucketName = "my-unique-bucket";
+   * Channel channel =
+   * 	Channel.newBuilder()
+   * 		.setKind("api#channel")
+   * 		.setId("channelId")
+   * 		.setResourceId("resourceId")
+   * 		.setAddress("address")
+   * 		.setPayload(true)
+   * 		.setType("WEBHOOK")
+   * 		.build();
+   * Channel response = storage.watchAll(bucketName, channel);
+   * }</pre>
+   *
+   * @return the instance of Channel
+   * @throws StorageException upon failure
+   * @see <a href="https://cloud.google.com/storage/docs/json_api/v1/objects/watchAll">watchAll</a>
+   */
+  Channel watchAll(String bucket, Channel channel, WatchAllOption... options);
+
+  /**
+   * Stops receiving object change notifications through the channel.
+   *
+   * <p>Example of stop receiving object change notifications through the channel.
+   *
+   * <pre>{@code
+   * String bucketName = "my-unique-bucket";
+   * Channel channel =
+   * 	Channel.newBuilder()
+   * 		.setKind("api#channel")
+   * 		.setId("channelId")
+   * 		.setResourceId("resourceId")
+   * 		.setAddress("address")
+   * 		.setPayload(true)
+   * 		.setType("WEBHOOK")
+   * 		.build();
+   * Channel response = storage.watchAll(bucketName, channel);
+   * boolean stopped = storage.stop(response);
+   * if (stopped) {
+   *   // the channel is stopped.
+   * } else {
+   *   // the channel was not found
+   * }
+   * }</pre>
+   *
+   * @return {@code true} if channel was stopped, {@code false} if it was not found
+   * @throws StorageException upon failure
+   * @see <a href="https://cloud.google.com/storage/docs/json_api/v1/channels/stop">stop</a>
+   */
+  boolean stop(Channel channel);
+
```

Comment:
'stop' is too generic name. Would you consider renaming
- watchAll -> startWathcAll
- stop  -> stopWatchAll

I know you took these names from com.google.api.services.storage.model.Channel, but in the Storage interface they look not very naturally.

Reply:
I think we need to go with [watchAll()](https://cloud.google.com/storage/docs/json_api/v1/objects/watchAll) and [stop()](https://cloud.google.com/storage/docs/json_api/v1/channels/stop) as per api doc.

Reply:
I do recommend to check the names with frankyn/elharo before submitting the PR.
Methods which sound normally for channel may be confusing for storage class.